### DISCUSSION TITLE
Is there any method to support nextra

### DISCUSSION THREAD

**Discussion by ChiaYuSu** [nextauthjs/next-auth#10925] [Help]
I want to use auth.js on nextra, is it possible?

**Comment by crguezl** [comment]
@ChiaYuSu  

NextAuth is already compatible with Nextra.  

I’ve written a tutorial for Nextra 2, which you can find here: [NextAuth Tutorial](https://ull-pl.vercel.app/nextra-playground/authorization/next-auth-tutorial).  
I’ve also tested it with Nextra 3, and it works with just a few minor adjustments.  

To fully support NextAuth in Nextra, it might simply require adding a few sections to the tutorial to explain its integration in detail.
